src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days <= 2) {
+    return days * 40;
+  } else if (days > 2 && days <= 6) {
```

Comment:
```suggestion
  } else if (days <= 6) {
```
Якщо перша перевірка пройшла, то до цього else if програма далі не дойде і немає сенсу перевіряти, чи днів більше 2, бо якщо перша НЕ пройшла, то їх точно не менше 2